mat64/symmetric_example_test.go

```diff
@@ -0,0 +1,49 @@
+package mat64_test
+
+import (
+	"fmt"
+
+	"github.com/gonum/matrix/mat64"
+)
+
+func ExampleSubsetSym() {
+	n := 5
+	s := mat64.NewSymDense(5, nil)
+	count := 1.0
+	for i := 0; i < n; i++ {
+		for j := i; j < n; j++ {
+			s.SetSym(i, j, count)
+			count++
+		}
+	}
+	fmt.Println("Original matrix:")
+	fmt.Printf("%0.4v\n", mat64.Formatted(s))
```

Comment:
Wandered off into commit land again.

Extra newline at end of each fmt.Printf format (except last).